File: 1074.py
N = int(input())
lista = []
for i in range(0, N):
    o = int(input())
    if o < 0:
        if o % 2 != 0:
            lista += ['ODD NEGATIVE']
        if o % 2 == 0:
            lista += ['EVEN NEGATIVE']
    if o == 0:
        lista += ['NULL']
    if o > 0:
        if o % 2 != 0:
            lista += ['ODD POSITIVE']
        if o % 2 == 0:
            lista += ['EVEN POSITIVE']
for w in range(0, N):
    print(f'{lista[w]}')

File: 1074.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var N = Number(lines.shift())
let lista = []
for (var i = 0;i < N; i++){
    o = Number(lines.shift());
    if(o < 0){
        if(o % 2 !== 0){
            lista.push('ODD NEGATIVE')}
        if(o % 2 === 0){
            lista.push('EVEN NEGATIVE')}}
    if (o === 0){
        lista.push('NULL')}
    if(o > 0){
        if(o % 2 !== 0){
            lista.push('ODD POSITIVE')}
        if (o % 2 === 0){
            lista.push('EVEN POSITIVE')}}}
for (var w = 0;w<N;w++){
    console.log(`${lista[w]}`)}
